main.js

```diff
@@ -0,0 +1,48 @@
+let today = new Date();
+let yyyy = today.getFullYear();
+document.querySelector("#copyrightYear").innerHTML = yyyy;
+
+let expandMenu = document.getElementsByClassName("open-menu");
+let arrow = document.getElementsByClassName("arrow-down");
```

Comment:
let is used for variables - so only if we plan to change its content at some point. If the value isn't going to be updated, stick to const.